File: clinica SPRING boot/demo/src/main/java/com/example/clinica/dtos/GetUsuarioDto.java
package com.example.clinica.dtos;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class GetUsuarioDto {
    private String dni;
    private String nombre;
    private String email;
    private String telefono;
    private String rol; 
}

File: clinica_app/lib/pages/data/GetUsuarioDto.dart
class GetUsuarioDto {
  final String dni;
  final String nombre;
  final String email;
  final String telefono;
  final String rol;

  GetUsuarioDto({
    required this.dni,
    required this.nombre,
    required this.email,
    required this.telefono,
    required this.rol,
  });

  factory GetUsuarioDto.fromJson(Map<String, dynamic> json) {
    return GetUsuarioDto(
      dni: json['dni'],
      nombre: json['nombre'],
      email: json['email'],
      telefono: json['telefono'],
      rol: json['rol'],
    );
  }
}